### PR TITLE
Update MartinezSensitivityAlgorithm_doc.i.in

### PR DESCRIPTION
**Pull request by mbaudin47**
Clarifies the dimension of the vectors z and z'

### CODE REVIEW

**Review by jschueller — APPROVED** [review]

**Review by sofianehaddad — APPROVED** [review]

**Comment by jschueller** [comment]
one quick review needed @JPelamatti, @josephmure or @m-balesdent

**Comment by mbaudin47** [comment]
> If z and z' are defined on RN, the pair (z,z') is defined on R2N

You are right, because the vector is in a cartesian product of two vector spaces. Should we write this $(\boldsymbol{z}, \boldsymbol{z}')^T \in \mathbb{R}^{2N}$ as you suggest or $(\boldsymbol{z}, \boldsymbol{z}')^T \in \left(\mathbb{R}^{N}\right)^2$? The second expression makes clear that this is the n-ary cartesian product $\mathbb{R}^{N} \times \mathbb{R}^{N}$ (see [Wikipedia](https://en.wikipedia.org/wiki/Cartesian_product#n-ary_Cartesian_power) on this topic).

**Comment by m-balesdent** [comment]
> > If z and z' are defined on RN, the pair (z,z') is defined on R2N
> 
> You are right, because the vector is in a cartesian product of two vector spaces. Should we write this (z,z′)T∈R2N as you suggest or (z,z′)T∈(RN)2? The second expression makes clear that this is the n-ary cartesian product RN×RN (see [Wikipedia](https://en.wikipedia.org/wiki/Cartesian_product#n-ary_Cartesian_power) on this topic).

Maybe " $\mathbb{R}^N×\mathbb{R}^N$"  can do the job.

File: python/src/MartinezSensitivityAlgorithm_doc.i.in [hunk 1]
```diff
@@ -52,6 +52,7 @@ sample has a zero mean) defined by:
    \frac{\sum_{k=1}^N z_k z_k'}
    {\sqrt{\sum_{k=1}^N z_k^2} \sqrt{\sum_{k=1}^N \left( z_k' \right)^2}}
 
+for any pair of vectors :math:`\vect{z}, \vect{z}' \in \Rset^N`
```
Comment (josephmure) [inline python/src/MartinezSensitivityAlgorithm_doc.i.in:55]: IMO this line you added should be moved up to line 47:

"... has a zero mean) defined for any pair of vectors :math:`\vect{z}, \vect{z}' \in \Rset^N` by: "

**Review by josephmure — CHANGES_REQUESTED** [review]

File: python/src/MartinezSensitivityAlgorithm_doc.i.in (hunk 1 above)
Comment (mbaudin47) [reply python/src/MartinezSensitivityAlgorithm_doc.i.in]: Ok : fixed at 1b5ad3206

**Comment by mbaudin47** [comment]
@josephmure : May I suggest to finalize this review?

**Review by josephmure — APPROVED** [review]

**Comment by mbaudin47** [comment]
@jschueller : I think this is ready for merging.

**Pull request merged**